README.md

```diff
@@ -53,3 +52,20 @@ spec:
     - name: tls
       mountPath: /tls
 ```
+
+`SecretClass` defines where the secrets come from. For example, the following `SecretClass`
```

Comment:
Can we add a list of currently available implementations and the config parameters?

Reply:
Done, but we'll probably want to reformat it at some point.